internal/controller/release_controller.go

```diff
@@ -249,7 +249,7 @@ func (r *ReleaseReconciler) reconcileHMCTemplates(ctx context.Context, releaseNa
 	if releaseName == "" {
 		releaseName = utils.ReleaseNameFromVersion(build.Version)
 		releaseVersion = build.Version
-		err := helm.ReconcileHelmRepository(ctx, r.Client, defaultRepoName, r.SystemNamespace, r.DefaultRegistryConfig.HelmRepositorySpec())
+		err := helm.ReconcileHelmRepository(ctx, r.Client, hmc.DefaultRepoName, r.SystemNamespace, r.DefaultRegistryConfig.HelmRepositorySpec())
```

Comment:
wdyt about keeping only helmrepository reference as a controller option and adding a default HelmRepository to the templates chart instead?

Reply:
will be discussed separately